DSharpPlus.Lavalink/LavalinkExtension.cs

```diff
@@ -18,14 +21,18 @@ public sealed class LavalinkExtension : BaseExtension
         }
         private AsyncEvent<NodeDisconnectedEventArgs> _nodeDisconnected;
 
-        private ConcurrentDictionary<ConnectionEndpoint, LavalinkNodeConnection> ConnectedNodes { get; }
+        /// <summary>
+        /// Gets a dictionary of connected Lavalink nodes for the extension.
+        /// </summary>
+        public IReadOnlyDictionary<ConnectionEndpoint, LavalinkNodeConnection> ConnectedNodes { get; }
```

Comment:
I am not so sure I like this. What's the rationale?

Reply:
It basically adds the ability to do LINQ operations on them, similar to other dictionaries we expose. For example, it can be useful for filtering based on player state for the connected guilds or for filtering by node statistics/connections for the connected nodes.

Reply:
But with nodes, isn't all of this achieved by your load balancing?

Reply:
Not necessarily. The load balancing method only returns the node with the least "penalties" in our eyes. Other users may want to calculate it a different way, or use the read only collections for other purposes. This is more of a convenience feature than anything else.

Reply:
I see.